ruff_dev/src/main.rs

```diff
@@ -1,3 +1,6 @@
+//! This crate implements an internal CLI for developers of ruff.
```

Comment:
This is a little bit of a nit but how are we choosing to stylize "ruff" in these various docs? In the docs, I tend to do "Ruff" when referring to the project by name and `ruff` when referring to the application binary.

Reply:
Makes sense ... changed to Ruff.